OpenEdXMobile/src/main/java/org/edx/mobile/player/PlayerController.java

```diff
@@ -233,7 +253,42 @@ private void initControllerView(View v) {
         installPrevNextListeners();
 
         mTitleTextView = (TextView) v.findViewById(R.id.video_title);
+        mRewindTimeTextView = (TextView) v.findViewById(R.id.rewind_time);
+        mForwardTimeTextView = (TextView) v.findViewById(R.id.forward_time);
         mTopBar = v.findViewById(R.id.video_top_bar);
+
+        initializeAnimations();
+    }
+
+    private void initializeAnimations() {
+        boolean isAnimLTR = (ViewCompat.getLayoutDirection(getRootView()) == ViewCompat.LAYOUT_DIRECTION_LTR);
+        mForwardAnimation = getSeekTimeAnimation(mForwardTimeTextView, isAnimLTR);
+        mRewindAnimation = getSeekTimeAnimation(mRewindTimeTextView, !isAnimLTR);
+    }
+
+    private TranslateAnimation getSeekTimeAnimation(TextView timeText, boolean isAnimLTR) {
+        TranslateAnimation translateAnimation = new TranslateAnimation(0.0f,
+                isAnimLTR ? ANIMATION_DISPLACEMENT : -ANIMATION_DISPLACEMENT, 0.0f, 0.0f);
+        translateAnimation.setDuration(DEFAULT_ANIMATION_DURATION_MS); // animation duration, change accordingly
+        translateAnimation.setInterpolator(new DecelerateInterpolator());
+        translateAnimation.setRepeatCount(0); // animation repeat count

```

Comment:
As per doc `The repeat count is 0 by default.` so better to remove this statement.